mod_wasm/modules/wasm/mod_wasm.c

```diff
@@ -104,14 +71,9 @@ typedef struct x_cfg {
                                                               * here?
                                                               */
     int congenital;                                          /* Boolean: did we inherit an "Example"? */
```

Comment:
Can we clean some attributes? I think some of these attributes came with the example, but I am not sure if they are critical for mod_wasm. We can also do the cleanup at a later time.

Reply:
Yes, you are absolutely right! I will get rid of them.